sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -1063,7 +1063,7 @@ class UnsupportedOperationsSuite extends SparkFunSuite with SQLHelper {
   /** Assert that the logical plan is supported as a batch plan */
   def assertSupportedInBatchPlan(name: String, plan: LogicalPlan): Unit = {
     test(s"batch plan - $name: supported") {
-      UnsupportedOperationChecker.checkForBatch(plan)
+      UnsupportedOperationChecker.checkForBatch(plan = plan)
```

Comment:
Why this change? It's pretty common to pass the variable whose name is the same as the function parameter name.